RollOver.js

```diff
@@ -9,7 +9,7 @@ function rollOver() {
         }
     }
 
-    console.log('The little one said, Everybody back on the bed!');
+    console.log('The little one Said, Everybody back on the bed!');
 }
```

Comment:
what is the purpose of assigning j = i ? could you not use i in places where you used j?